### DISCUSSION TITLE
Unable to login to SOC

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#9689] [Unsupported Versions] [answered]
I am unable to login to the SOC after rebooting my so appliance. It is telling me "The provided credentials are invalid. Please try again". I had already created 2 accounts for the dashboard that worked prior to reboot. 

so-user-list shows no users. The logs for sensoroni and soc do not show anything.

I am on the latest version of SO, 2.3.200. This was a fresh installation.

**Comment by InfosecGoon** [comment]
If you add a user with "so-user-add" and then run "so-user-list" again, does it show the new account?

After adding the new user, are there any errors in /opt/so/log/kratos/kratos.log?

**Reply by thegreekman** [reply]
Thanks for the reply - i was able to resolve the issue

**Comment by thegreekman** [comment, marked as answer]
The cause of the problem was that I was mounting the /nsm drive after boot, manually. Although all the services were successful and online, something else was happening on the backend. So-user-list would return no accounts, and when I made a test account, it WOULD show that account. I resolved the issue by having the /nsm drive mount during boot. The original accounts came back, and the test account was no longer there.

The reason I had to mount /nsm after boot was because it was attached to an iscsi driver, which requires a network connection to mount. So if I left the mount-point for nsm in my /etc/fstab file, the device wouldn't boot because that drive did not exist yet. After learning you can actually auto-mount iscsi storage after NETWORK boot, it fixed my problem.
